test/test_chat_interface.py

```diff
@@ -351,3 +351,12 @@ def test_setup_example_messages_empty(self):
         chat = gr.ChatInterface(double)
         chat._setup_example_messages(None)
         assert chat.examples_messages == []
+
+    def test_chat_interface_disable_api(self, connect):
```

Comment:
very nit: since you also test changing the api_name to "double" here
```suggestion
    def test_chat_interface_api_name(self, connect):
```